golang/cosmos/x/vibc/keeper/keeper.go

```diff
@@ -92,13 +92,7 @@ func (k Keeper) ChanOpenInit(ctx sdk.Context, order channeltypes.Order, connecti
 		return err
 	}
 
-	// We need to emit a channel event to notify the relayer.
-	ctx.EventManager().EmitEvents(sdk.Events{
-		sdk.NewEvent(
-			sdk.EventTypeMessage,
-			sdk.NewAttribute(sdk.AttributeKeyModule, channeltypes.AttributeValueCategory),
-		),
-	})
+	k.channelKeeper.WriteOpenInitChannel(ctx, portID, channelID, order, connectionHops, counterparty, version)
```

Comment:
@michaelfig please verify that this is good to include.

Reply:
Yes, please!  This is a critical platform fix whose absence would prevent the use of dIBC in JS contracts.